app/models/subreddit.rb

```diff
@@ -0,0 +1,26 @@
+class Subreddit
+
+  attr_accessor :rules
+
+  attr_reader :name
+
+  def initialize(name, token)
+    @name = name
+    @rules = Array.new
+    @token = token
+  end
+
+  def get_rules
+    serv = RedditService.new(@token)
```

Comment:
@CjMoore Can you talk more about what makes you uncomfortable with this? Regarding the token, I think that you have the option of passing it directly to the method, but the idea of storing it on the PORO doesn't bother me that much. There's a little bit of complexity here, but the `#show` method on the `SubredditController` looks very clean, so I don't think it bothers me. Let me know if you'd like to talk tomorrow, and maybe we can think of a better approach. 👍